### ISSUE TITLE
Enabling visual helper in Keyboard or Focus assessment brings the target page to the front but the tab stops visualization doesn't work

### ISSUE THREAD

**Issue by AhmedAbdoOrtiga** [microsoft/accessibility-insights-web#1111]
## Describe the bug

Enabling visual helper Keyboard navigation brings the target page to the front but the tab stops visualization doesn't work **Only if the target page is minimized**

**To Reproduce**

1. Go to any web page.
2. Start new assessment.
3. Switch to Keyboard or Focus assessment. 
4. **Minimize the target page.**
5. Enable visual helper.
6. Starting clicking tab.
7. Tab stops visualization doesn't work.

## Expected behavior

Tab stops visualization should work.

## Extension (please complete the following information)

- OS: [Windows]
- Version [2.8.0]
- Browser Version [Chrome 76.0.3809.100]
- Target Page [any page eg: www.microsoft.com]

**Comment by waabid** [comment]
A couple of notes:

- The target page is un-minimized when the tab stops experience is enabled (this was a bit unclear to me reading the bug description).
- Upon investigation, even when the target page is un-minimized the page does not receive focus. If one were to manually focus on the page, the visualization does work (meaning code is injected appropriately).
